src/components/Accordion.js

```diff
@@ -47,8 +47,8 @@ const Accordion = React.createClass({
         Svg("arrow", { className: opened[i] ? "expanded" : "" }),
         item.header,
         item.buttons ?
-        dom.button({ className: "header-buttons" },
-          item.buttons.map((button, id) => span({ key: id }, button))
+        dom.span({ className: "header-buttons" },
```

Comment:
is this fine semantically? or should we use a div?

@wldcordeiro, @Garbee advice?